src/pint/toa.py

```diff
@@ -143,6 +150,27 @@ def get_TOAs(
         Completed TOAs object representing the data.
 
     """
+    if model:
+        # If the keyword args are set, override what is in the model
+        if ephem is None and model["EPHEM"].value is not None:
+            ephem = model["EPHEM"].value
+            log.info(f"Using EPHEM = {ephem} from the given model")
+        if include_bipm is None and model["CLOCK"].value is not None:
+            if "BIPM" in model["CLOCK"].value:
```

Comment:
Is it worth emitting a warning if something went wrong - if `CLOCK` is set but we don't know what to do with it?

Reply:
Yeah, that's a good idea.  I'll do that.  Right now, if a CLOCK line is not TT(BPIMXXXX) we basically don't do anything with it.